plugins/modules/powervm_lpar_migration.py

```diff
@@ -96,6 +96,28 @@
             - The maximum time, in minutes, to wait for operation to complete
             - This option can be used only with C(migrate) and C(validate) I(action)
         type: int
+    shared_proc_pool:
```

Comment:
mention "this parameter support single and multiple pools"
also mention what all actions it supports.